ajax/libs/jquery-flexdatalist/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery-flexdatalist",
+  "version": "1.9.5",
+  "description": "jQuery Datalist Autocomplete plugin",
+  "keywords": [
+    "jQuery",
+    "autocomplete",
+    "plugin",
+    "datalist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sergiodlopes/jquery-flexdatalist.git"
+  },
+  "author": "Sérgio Dinis Lopes",
+  "license": "MIT",
+  "homepage": "https://projects.sergiodinislopes.pt/flexdatalist/",
+  "npmName": "jquery-flexdatalist",
```

Comment:
@sophie-li https://www.npmjs.com/package/jquery-flexdatalist not found. Please transfer to use git auto-update config. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md). Thanks.